test/scaffold/scaffold.go

```diff
@@ -70,25 +70,32 @@ func newScaffold(conf cmdconfig.ClientConfig) *Scaffold {
 
 	ginkgo.AfterEach(func() {
 		for route := range s.routes {
-			s.DeleteRoute(route)
+			err := s.DeleteRoute(route)
```

Comment:
It is expected to ignore the errors. This can simply make sure all the delete operations are executed

Reply:
got it